src/routes/inngang/inngang-fullmakt.tsx

```diff
@@ -0,0 +1,138 @@
+import { AlertStripeFeil } from 'nav-frontend-alertstriper';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import Lenkepanel from 'nav-frontend-lenkepanel';
+import { FnrInput, Label } from 'nav-frontend-skjema';
+import React, { useState } from 'react';
+import styled from 'styled-components/macro';
+import { useBreadcrumbs } from '../../breadcrumbs/use-breadcrumbs';
+import { InngangKategori, Kategori } from '../../kategorier/kategorier';
+import { usePageInit } from '../../page-init/page-init';
+import { MarginTopContainer } from '../../styled-components/common';
+import { ContentContainer } from '../../styled-components/content-container';
+import { InngangMainContainer } from '../../styled-components/main-container';
+import { CenteredPageTitle } from '../../styled-components/page-title';
+import { SectionTitle } from '../../styled-components/section-title';
+import { WhiteSection } from '../../styled-components/white-section';
+import { User } from '../../user/user';
+import { getFullName } from '../klageskjema/summary/personlige-opplysninger-summary';
+import { foedselsnrFormat } from '../klageskjema/summary/text-formatting';
+import queryString from 'query-string';
+import { hasFullmaktFor } from '../../api/api';
+
+interface Props {
+    inngangkategori: InngangKategori;
+    kategori: Kategori;
+}
+
+const FieldWithButton = styled.div`
+    display: flex;
+    flex-flow: row wrap;
+    justify-items: space-around;
+    > * {
+        flex-basis: 180px;
+        margin: 10px 0;
+    }
+    justify-content: flex-start;
+`;
+
+const InngangFullmakt = ({ inngangkategori, kategori }: Props) => {
+    const { title, temaKey } = kategori;
+    usePageInit(`${title} \u2013 klage på vegne av andre`);
+    useBreadcrumbs([], 'Klage på vegne av andre');
+
+    const [fodselsnummer, setFodselsnummer] = useState<string | number | readonly string[] | undefined>('');
+    const [valid, setValid] = useState<boolean>(false);
+    const [submit, setSubmit] = useState<boolean>(false);
+    const [loading, setLoading] = useState<boolean>(false);
+    const [feilmelding, setFeilmelding] = useState<string>('');
+    const [selectedFullmaktsgiver, setSelectedFullmaktsgiver] = useState<User | null>(null);
+
+    const query = queryString.stringify(
+        {
+            tema: temaKey,
+            tittel: title,
+            fullmaktsgiver: fodselsnummer
+        },
+        {
+            skipNull: true
+        }
+    );
+
+    const handleSubmit = async (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        console.log(e);
+        setSubmit(true);
+        setLoading(true);
+        setFeilmelding('');
+        setSelectedFullmaktsgiver(null);
+        if (temaKey !== null && fodselsnummer) {
```

Comment:
Dersom `fodselsnummer` får typen `string |  null` kan sjekken her være lik for `temaKey` og `fodselsnummer`.
`fodselsnummer !== null`